packages/base/lua/Event.lua

```diff
@@ -189,7 +189,7 @@ function Event(data)
 		}
 	end
 
-	if belong(type(data.action), {"Chart", "Map", "InternetSender", "VisualTable", "Clock", "FileSystem", "TextScreen"}) then
```

Comment:
Oops, there is something wrong here. `FileSystem` is not a type. This should be `LogFile`!